qiskit/algorithms/optimizers/de.py

```diff
@@ -0,0 +1,134 @@
+    have been reached or the standard deviation of the generation of candidates
+    is below a user-specified threshold. Specificially, the total number of
+    function evaluations equals (maxiter + 1)* popsize * len(x), without polishing
+    (see below).
+
+    Uses scipy.optimize.differential_evolution
+    For further detail, please refer to
+    https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.differential_evolution.html
+    """
+
+    _OPTIONS = ['maxiter', 'strategy', 'popsize', 'mutation',
+                'recombination', 'init', 'tol', 'atol',
+                'polish', 'disp', 'workers']
+
+    def __init__(self,
+                 maxiter: int = 10,
+                 strategy: str = 'currenttobest1bin',
+                 popsize: int = 10,
+                 mutation: Union[float, tuple] = (0.5, 1),
+                 recombination: float = 0.7,
+                 init: str = 'latinhypercube',
+                 tol: float = 0.01,
+                 atol: float = 0,
+                 polish: bool = False,
+                 disp: bool = False,
+                 workers: Union[int, Callable] = 1) -> None:
+        """
+        Args:
+            maxiter: Maximum number of generations to evolve population
+            strategy: Differential evolution strategy to employ for generating new trial candidates
+                      in subsequent generations/iterations
+            popsize: Number of individuals/candidates in each generation
+            mutation: Mutation constant (differential weight) specified as a float in the range of
+                      [0,2] or as a tuple(min, max). If specified as a tuple, the mutation constant
+                      will be 'dithered' or randomly changed within the range specified in each
+                      generation.
+            recombination: Recombination constant (crossover probability) specified as a float
+                           in the range [0,1]
+            init: String/array to specify whether initial population is generately via ``random``,
+                  ``latinhypercube`` sampling or via a specified array of individuals. If an array
+                  is specified, its shape must match (popsize, params) where popsize and params
+                  refer to population size and number of parameters in the objective function
+            tol: Relative tolerance for convergence
+            atol: Absolute tolerance for convergence
+            polish: Boolean to specify whether or not best population in last generation is
+                    optimized using scipy.optimize.minimize with the L-BFGS-B method. If
+                    this is set to true, the maximum number of function evaluations will
+                    be greater than (maxiter + 1)* popsize * len(x)
+            disp: Boolean to specify if the objective function is printed at each iteration
+            workers: Integer or map-like callable (multiprocessing.Pool.map) that specifies

```

Comment:
Have you tried this with more than one worker across processes etc? I was noting that the scipy docs for the function used has n note about func - the objective_function here - being pickleabe.